### ISSUE TITLE
Handle error if the user did download the wrong link 

### ISSUE THREAD

**Issue by EssamSoft** [tanhakabir/SwiftAudioPlayer#146]
it will be great if support handles errors like: 

```
  SAPlayer.Downloader.downloadAudio(withRemoteUrl: urlAudio, completion: { [weak self] (url, error) in
    guard let self = self else { return }
    if let error = error {
      print("can't download audio")
    }
  })
```

**Comment by AndreyPoznyak** [comment]
@tanhakabir  hey, any luck on this one? :) maybe there is some object I can observe the state of?

**Comment by tanhakabir** [comment]
Sorry working on it now!

**Comment by AndreyPoznyak** [comment]
would u suggest some quick workaround here? I understand it might take time to come up with proper implementation
I need to be able to handle http error when streaming the audio from remote

**Comment by tanhakabir** [comment]
Sorry I was on vacation! It's a simple thing to make I just haven't had a chance to get to it yet. Sorry I don't think there's an easy work around right now but I should have this in this week! So sorry!

**Comment by AndreyPoznyak** [comment]
gotcha, appreciated! @tanhakabir

**Comment by tanhakabir** [comment]
Released on version 7.5.0 (publishing now)

**Comment by AndreyPoznyak** [comment]
great, thanks
would there be any way to handle the error when we play remote stream? @tanhakabir 
via this method: `startRemoteAudio(withRemoteUrl: , mediaInfo: )`
